### ISSUE TITLE
About testset

### ISSUE THREAD

**Issue by katherinelyx** [thu-coai/KdConv#2]
Hi, congratulations for the acceptance of your ACL paper and thank you for your contributions.
I noticed that there was only training data available, do you have official test set?

**Comment by chujiezheng** [comment]
Thanks for your interest. We reserve the development and test sets for the follow-up competition. Thus we only release the training sets at present.

**Comment by katherinelyx** [comment]
Thank you for your nice reply.
When will the competition start?

**Comment by chujiezheng** [comment]
It is in preparation :) Once the competition is officially ready, we will issue a notice.

**Comment by katherinelyx** [comment]
Thank you.
